### PR TITLE
Issue 15

### PR DESCRIPTION
**Pull request by ptrimble**
For issue https://github.com/sittercity/gocumber/issues/15

### CODE REVIEW

File: gocumber_test.go [hunk 1]
```diff
@@ -140,11 +140,60 @@ func TestRun_SuccessWithOutlineSteps(t *testing.T) {
...
+		expected_first_example,
+		expected_second_example,
+		expected_third_example,
+		expected_fourth_example,
+		expected_fifth_example,
+	}
+
+	var called int
+	steps.When("I create a new user with the following data:", func(_ []string, step StepNode) {
+		assert.Equal(t, expected_replaced_table_data[called], ColumnMap(step.Table()))
```
Comment (cbandy) [inline gocumber_test.go]: This looks really good, but the expectations are a little far away for my taste. Was it too busy with the list of values here?

In the past I've used a `switch` to make mocks behave differently during different calls, e.g.

```go
switch called++; called {
  case 1:
    assert.Equal(t, map{
      ...,
    }, ColumnMap(step.Table()))
  case 2:
    assert.Equal(...)
  ...
}
```

**Comment by cbandy** [comment]
:thumbsup:

File: gocumber_test.go (hunk 1 above)
Comment (ptrimble) [inline gocumber_test.go]: Sure, no worries. Addressed in https://github.com/sittercity/gocumber/commit/b2e3703e941199b22a3afdaf490724191ba60090

**Comment by cbandy** [comment]
:thumbsup:

**Pull request merged**